File: crates/fresh-editor/tests/fixtures/syntax_highlighting/hello.lua
#!/usr/bin/env lua

-- Greeting function with string formatting
function greet(name)
    return string.format("Hello, %s!", name)
end

-- Table with mixed content
local config = {
    version = "1.0",
    enabled = true,
    count = 42
}

-- Main execution with conditional logic
if arg then
    local message = greet("World")
    print(message)

    for i = 1, 3 do
        print("Iteration: " .. i)
    end
end

File: crates/fresh-editor/tests/fixtures/syntax_highlighting/hello.pl
#!/usr/bin/env perl
use strict;
use warnings;

sub greet {
    my ($name) = @_;
    return "Hello, $name!";
}

my $message = greet("World");
print "$message\n";

my @items = (1, 2, 3);
my %config = (key => "value", count => 42);
